packages/core/admin/admin/src/content-manager/services/documents.ts

```diff
@@ -96,6 +96,35 @@ const documentApi = contentManagerApi.injectEndpoints({
       invalidatesTags: (_res, _error, { model, ids }) =>
         ids.map((id) => ({ type: 'Document', id: `${model}_${id}` })),
     }),
+    discardDocument: builder.mutation<
+      Contracts.CollectionTypes.Discard.Response,
+      Pick<Contracts.CollectionTypes.Discard.Params, 'model'> & {
+        collectionType: string;
```

Comment:
Should we add collectionType to the CM contract and make id optional there?

Reply:
I don't think so, i think the contracts are reflective of the API routes, meanwhile here i use one set of APIs for single & collection types